liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/RunServerMojo.java

```diff
@@ -45,12 +45,18 @@ protected void doExecute() throws Exception {
             getLog().info("\nSkipping run goal.\n");
             return;
         }
-        
+        String projectPackaging = project.getPackaging();
+
         runMojo("org.apache.maven.plugins", "maven-compiler-plugin", "compile");
+
+        if(!looseApplication && projectPackaging.equals("ear")) {
+            runMojo("org.apache.maven.plugins", "maven-ear-plugin", "generate-application-xml");
```

Comment:
Wondering if this should be done even if it is looseApplication? Dev mode forces loose app right? And it always runs this goal for the `ear` packaging type.

Reply:
Yeah, looking into it, looks like it's probably needed as well for looseApp. Will change it 👍